apisix/plugins/jwt-auth.lua

```diff
@@ -277,18 +285,44 @@ function _M.rewrite(conf, ctx)
     if not consumer then
         return 401, {message = "Invalid user key in JWT token"}
     end
-    core.log.info("consumer: ", core.json.delay_encode(consumer))
+    log.info("consumer: ", core.json.delay_encode(consumer))
 
     local _, auth_secret = algorithm_handler(consumer)
     jwt_obj = jwt:verify_jwt_obj(auth_secret, jwt_obj)
-    core.log.info("jwt object: ", core.json.delay_encode(jwt_obj))
+    log.info("jwt object: ", core.json.delay_encode(jwt_obj))
 
     if not jwt_obj.verified then
         return 401, {message = jwt_obj.reason}
     end
 
     consumer_mod.attach_consumer(ctx, consumer, consumer_conf)
-    core.log.info("hit jwt-auth rewrite")
+    log.info("hit jwt-auth rewrite")
+end
+
+
+local function user_info()
```

Comment:
Is this function necessary for the core feature? We should only accept the necessary API to avoid bloating.

Reply:
Yes,the function provide ability to check up the payload content.

Reply:
This feature can be implemented as a local script to check.
There is no need to expose it as a public API of the gateway.

Reply:
I get it,thanks.